classes/requests/helpers/class-kp-order-lines.php

```diff
@@ -342,6 +342,30 @@ private function process_coupons() {
 				}
 			}
 		}
+
+		/**
+		 * PW WooCommerce Gift Cards.
+		 */
+		if ( ! empty( WC()->session->get( 'pw-gift-card-data' ) ) ) {
```

Comment:
I'm not sure if its absolutely required, but could we add a check to make sure that WC()->session is not null and actually exists?